examples/cloudwatch_listener_ex.py

```diff
@@ -0,0 +1,25 @@
+from locust import main, events
+from locust_plugins.listeners.cloudwatch import CloudwatchAdapter, ServiceContext
+from locust_plugins import missing_extra
+
+try:
+    import boto3
+except ModuleNotFoundError:
+    missing_extra("boto3", "boto3")
+
+
+@events.init.add_listener
+def on_locust_init(environment, **_kwargs):
+    CloudwatchAdapter(_cloudwatch(), environment, _service_context())
+
+
+def _cloudwatch():
+    return boto3.client("cloudwatch")
+
+
+def _service_context():
+    return ServiceContext("MyExampleService", "perf")
+
+
+if __name__ == "__main__":
```

Comment:
this is non-standard. sometimes we do a run_single_user if the file is run directly, but never main.main()

Reply:
This is for running locust with all the locust features after I register the listener. Don't want to run a single user case. Please suggest how I can do that with making this call. I will fix it accordingly.

Reply:
You can just remove it entirely if you dont want to do run_single_user.

Reply:
How do we let locust know that we want register this listener before starting the performance test? Basically how do I bootstrap my code into locust execution. Please let me know so that I can learn and fix it.

Reply:
Not sure exactly what you mean. The init listeners get called before the test starts, regardless if you run the test using `locust -f cloudwatch_listener_ex.py` or using `run_single_user`. Locust files are not typically executed using `python cloudwatch_listener_ex.py`, except when debugging using `run_single_user`

Reply:
I got it now. Will make the change rightaway.

Reply:
Completed the change.